Gems/RemoteTools/Code/Source/RemoteToolsSystemComponent.cpp

```diff
@@ -563,15 +563,6 @@ namespace RemoteTools
             m_joinThread->Join();
             m_joinThread->Start();
         }
-
-        // Check if there are any active connections, if not stop the outbox thread
-        bool hasActiveConnection = false;
-        for (auto registryIt = m_entryRegistry.begin(); registryIt != m_entryRegistry.end(); ++registryIt)
-        {
-            hasActiveConnection =
-                AZ::Interface<AzNetworking::INetworking>::Get()->RetrieveNetworkInterface(registryIt->second.m_name) != nullptr;
-        }
```

Comment:
This query is not necessary anymore because before it used to do

````
        if (!hasActiveConnection)
        {
            m_outboxThread->Stop();
            m_outboxThread->Join();
        }
````

But not anymore since this change: https://github.com/o3de/o3de/pull/15112